test/extended/networking/util.go

```diff
@@ -95,10 +95,43 @@ func expectError(err error, explain ...interface{}) {
 	ExpectWithOffset(1, err).To(HaveOccurred(), explain...)
 }
 
-func launchWebserverService(client k8sclient.Interface, namespace, serviceName string, nodeName string) (serviceAddr string) {
+func launchWebserverNodePortService(client k8sclient.Interface, namespace, serviceName string, nodeName string) int32 {
```

Comment:
This is no longer needed in `networking/util.go` since you copied the functions out into `kubevirt/util.go` (except that you shouldn't use them there either :slightly_smiling_face: )